pgmpy/inference/ExactInference.py

```diff
@@ -171,6 +171,10 @@ def map_query(self, variables=None, evidence=None, elimination_order=None):
         """
         Computes the MAP Query over the variables given the evidence.
 
+        TODO: Change this behavior to return the joint MAP
```

Comment:
@ankurankan Please put TODO in inline comments rather than docstring.

Reply:
@khalibartan But why ? This seems to be the best place to put this todo.

Reply:
@ankurankan Usual standard is to use #TODO (even IDE support identification of #TODO) rather than putting TODO in docstring. I think this is because TODO are for devs not for users.

Reply:
@khalibartan Done.